### PR TITLE
Update new release branch procedure

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/developer/build_release.rst.tmpl [hunk 1]
```diff
@@ -52,99 +49,97 @@ On the demo you should create the new version branch.
...
-``<new version>``.
-
-On the master branch, in the file ``.github/workflows/ci.yaml`` set ``MAJOR_VERSION`` to
-``<next version>``.
-
-Create the new Transifex resources
------------------------------------
-
-On ngeo checkout the new version branch
+On the demo You should set the default branch th the new branches.
```
Comment (ger-benjamin) [inline doc/developer/build_release.rst.tmpl:54]:
```suggestion
On the demo you should set the default branch to the new branch.
```

File: doc/developer/build_release.rst.tmpl [hunk 2]
```diff
@@ -52,99 +49,97 @@ On the demo you should create the new version branch.
...
 
-.. prompt:: bash
+On de demo new branche you should copy the file ``.github/workflows/upgrade-<new version>.yaml`` to
```
Comment (ger-benjamin) [inline doc/developer/build_release.rst.tmpl:56]:
```suggestion
On the new branch of the demo you should copy the file ``.github/workflows/upgrade-<new version>.yaml`` to
```

File: doc/developer/build_release.rst.tmpl (hunk 1 above)
Comment (arnaud-morvan) [inline doc/developer/build_release.rst.tmpl:54]:
```suggestion
On the demo GitHub repository you should set the default branch to the new branch.
```

File: doc/developer/build_release.rst.tmpl (hunk 2 above)
Comment (arnaud-morvan) [inline doc/developer/build_release.rst.tmpl:56]:
```suggestion
On the demo new branch you should copy the file ``.github/workflows/upgrade-<new version>.yaml`` to
```

File: doc/developer/build_release.rst.tmpl [hunk 3]
```diff
@@ -258,18 +260,47 @@ On the ``c2cgeoportal`` new version branch:
...
+
+Backport label
+--------------
+
+Create the new backport label named ``backport_<new_version>``.
+
+Protect branch
+--------------
+
+In GitHub project settings, pritect the new branch with the same settings as the master branch.
```
Comment (ger-benjamin) [inline doc/developer/build_release.rst.tmpl:290]:
```suggestion
In GitHub project settings, protect the new branch with the same settings as the master branch.
```

File: doc/developer/build_release.rst.tmpl [hunk 4]
```diff
@@ -258,18 +260,47 @@ On the ``c2cgeoportal`` new version branch:
...
+
+Check
+-----
+
+Run `c2cciutils-checks` on each branche bepor pushing to be sure that everything is OK.
```
Comment (ger-benjamin) [inline doc/developer/build_release.rst.tmpl:295]:
```suggestion
Run `c2cciutils-checks` on each branch before pushing to be sure that everything is OK.
```

**Review by ger-benjamin — APPROVED** [review]

File: doc/developer/build_release.rst.tmpl [hunk 5]
```diff
@@ -258,18 +260,47 @@ On the ``c2cgeoportal`` new version branch:
        ngeo: {}
      releases: []
 
+Security information
+--------------------
+
+On the master branch the file ``SECURITY.md`` with the security information by adding:
```
Comment (arnaud-morvan) [inline doc/developer/build_release.rst.tmpl:266]:
```suggestion
On the master branch, update the file ``SECURITY.md`` with the security information by adding:
```

File: doc/developer/build_release.rst.tmpl (hunk 3 above)
Comment (arnaud-morvan) [inline doc/developer/build_release.rst.tmpl:290]:
```suggestion
In GitHub project settings, protect the new branch with the same settings as the master branch.
```

File: doc/developer/build_release.rst.tmpl (hunk 4 above)
Comment (arnaud-morvan) [inline doc/developer/build_release.rst.tmpl:295]:
```suggestion
Run `c2cciutils-checks` on each branch before pushing to be sure that everything is OK.
```

**Review by arnaud-morvan — APPROVED** [review]
LGTM, with some typos.

**Pull request merged**